src/components/TimelineContainer.jsx

```diff
@@ -0,0 +1,74 @@
+import requestHandler from "../js/requestHandler.js";
+import TweetContainer from "./TweetContainer.jsx";
+
+class TimelineContainer extends React.Component {
+    constructor(props) {
+        super(props);
+
+        this.state = {
+            data: [],
+            errorOccurred: false
+        }
+
+        this.updateStatus = (responseStatus, responseData) => {
+            if(responseStatus === 200) {
+                this.setState({
+                    data: JSON.parse(responseData),
+                    errorOccurred: false
+                });
+            } else if (responseStatus !== 200) {
+                this.setState({
+                    data: [],
+                    errorOccurred: true
+                });
+            }
+        }
+    }
+
+    componentDidMount() {
+        requestHandler(this.updateStatus);
+    }
+
+    render() {
+        let timelineContainer;
+        if(this.state.errorOccurred === true) {
+            timelineContainer = (
+                <div id="timelineContainer">
+                    <p>This content is not currently available. Please try again later.</p>
+                </div>
+            );
+        } else if(this.state.errorOccurred === false && this.state.data.length > 0) {
+            timelineContainer = (
+                <div id="timelineContainer">
+                    {this.state.data.map(status => 
+                        <TweetContainer key={status.postUrl} user={status.user} postUrl={status.postUrl} message={status.message} createdAt={status.createdAt}/>
+                    )}
+                </div> 
+            );
+        } else if (this.state.errorOccurred === false) {
+            timelineContainer = (
+                <div id="timelineContainer">
+                    <p>Loading your Twitter timeline...</p>
+                </div>
+            );
+        }
+
+        return (
+            <div id="apiContainer">
+                <button id="apiButton" type="button" onClick={
+                    (e) => {
```

Comment:
i'd recommend putting the functions into functions and then assigning them in the constructor.  It makes it more readable and easier to search for as well